cars/v1/parallelgc.ini

```diff
@@ -9,3 +9,4 @@ base=vanilla
 use_cms_gc=false
 use_g1_gc=false
 use_parallel_gc=true
+use_zgc=false
```

Comment:
I don't understand why all GC mixins have all these `false` variables. But if we add it here, can you also add it to g1gc.ini?

Reply:
TBH, it would work fine without defined variables. I only added them to be consistent with the existing templates. Perhaps deleting them from other templates instead would make more sense.

Reply:
Removed the variables instead.